index.bs

```diff
@@ -626,14 +647,264 @@ interface MLContext {};
+                1. Else:
+                    1. For each |dimension| of |inputDesc|.{{MLOperandDescriptor/dimensions}}:
+                        1. The value of |dimension| must be greater than 0.
+                        1. Set |inputSize| to the product of |inputSize| and |dimension|.
+                1. If |value| is an {{MLArrayInput}}, then let |resource| be |value|.{{MLArrayInput/resource}}.
+                1. If |resource| is an {{ArrayBufferView}}, then:
+                    1. The kind of |resource| must be compatible with |inputDesc|.{{MLOperandDescriptor/type}} according to [this table](#appendices-mloperandtype-arraybufferview-compatibility).
+                    1. The length of |resource| must be the same as |inputSize|.
+            1. For each |key| -> |value| of |outputs|:
+                1. |graph|.{{MLGraph/[[outputNames]]}}[|key|] must exist.
+        </div>
+    1. For each |key| -> |value| of |inputs|:
+        1. Let |inputDesc| be |graph|.{{MLGraph/[[inputDescriptors]]}}[|key|].
+        1. Let |inputTensor| be a new tensor for |graph|.{{MLGraph/[[implementation]]}} of data type that is compatible with |inputDesc|.{{MLOperandDescriptor/type}}.
+        1. If |value| is an {{MLArrayInput}}, then:
+            1. Set the dimensions of |inputTensor| to |value|.{{MLArrayInput/dimensions}}.
+        1. Else:
+            1. Set the dimensions of |inputTensor| to |inputDesc|.{{MLOperandDescriptor/dimensions}}.
+        1. If |value| is an {{MLArrayInput}}, then:
+            1. Set the values of |inputTensor| to the values of |value|.{{MLArrayInput/resource}}.
+        1. Set the input of |graph|.{{MLGraph/[[implementation]]}} that is associated with |key| to |inputTensor|.
+    1. For each |key| -> |value| of |outputs|:
+        1. Issue a compute request for output of |graph|.{{MLGraph/[[implementation]]}} that is associated with |key|.
+        1. Wait for the compute request to be completed.
+        1. If there is an error returned by |graph|.{{MLGraph/[[implementation]]}}, then:
+            1. Throw an {{OperationError}} {{DOMException}} and stop.
+        1. Else:
+            1. Let |outputTensor| be the output tensor returned by |graph|.{{MLGraph/[[implementation]]}}.
+            1. If the kind of |value| is not compatible with the value type of |outputTensor|, then throw a {{DataError}} {{DOMException}} and stop.
+            1. Let |outputSize| be 1.
+            1. For each |dimension| of dimensions of |outputTensor|:
+                1. Set |outputSize| to the product of |outputSize| and |dimension|.
+            1. If |outputSize| is greater than the length of |value|, then:
+                1. Throw a {{DataError}} {{DOMException}} and stop.
+            1. Else:
+                1. Set the values of |value| to the values of |outputTensor|.
+    1. Return Promise<{{undefined}}>.
+</div>
+
+<div class="note">
```

Comment:
This note section should be removed because the `computeAsync` method is now only allowed in an `MLContext` created with `MLContextOptions`.

Reply:
Good point. Will do.